### PR TITLE
Add dockerfiles for glitch

### PR DESCRIPTION
**Pull request by halfwit**
More docker things, this one for glitch

### CODE REVIEW

File: glitch/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    environment:
+      MYSQL_ROOT_PASSWORD: $DB_PASSWORD
+      MYSQL_DATABASE: $DB_DATABASE 
+      MYSQL_USER: $DB_USERNAME 
+      MYSQL_PASSWORD: $DB_PASSWORD
+    volumes:
+      - ./schema.sql:/docker-entrypoint-initdb.d/setup.sql
+      - dbdata:/var/lib/mysql
+    networks:
+      - glitch-network
```
Comment (itsrachelfish) [inline glitch/docker-compose.yml]: Glitch doesn't use a database, it's purely PHP + HTML. No point in running a database for this project

**Review by itsrachelfish — CHANGES_REQUESTED** [review]
This looks good, but the MySQL portions need to be removed.

File: glitch/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+#Docker Networks
+networks:
+  glitch-network:
+    driver: bridge
+
+#Volumes
+volumes:
+    dbdata:
+      driver: local
```
Comment (itsrachelfish) [inline glitch/docker-compose.yml]: Shouldn't this database volume also be removed?

Comment (halfwit) [reply glitch/docker-compose.yml]: They definitely don't need to be there, my bad. I'll edit in a sec

**Review by itsrachelfish — APPROVED** [review]
It works :sunglasses:

**Pull request merged**